### PR TITLE
Unlicensed hull block

### PR DESCRIPTION
**Pull request by ghost**
Not sure why it says I altered the construction costs.... I didn't touch those.

### CODE REVIEW

File: Data/CubeBlocks.sbc [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0"?>
-<Definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```
Comment (zrisher) [inline Data/CubeBlocks.sbc:2]: Any reason for removing the namespace attributes?

File: Data/CubeBlocks.sbc [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zrisher) [inline Data/CubeBlocks.sbc:1]: :+1:

File: Data/CubeBlocks.sbc [hunk 3]
```diff
@@ -27,20 +27,15 @@
...
-        <MountPoint Side="Front" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-        <MountPoint Side="Back" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-      </MountPoints>
-      <BuildProgressModels>
-        <Model BuildPercentUpperBound="0.33" File="Models\Cubes\Large\BeaconConstruction_1.mwm" />
-        <Model BuildPercentUpperBound="0.66" File="Models\Cubes\Large\BeaconConstruction_2.mwm" />
-        <Model BuildPercentUpperBound="1.00" File="Models\Cubes\Large\BeaconConstruction_3.mwm" />
-      </BuildProgressModels>
-      <BlockPairName>GardenConquest_UnlicensedHullClassifier</BlockPairName>
+			<BuildProgressModels>
```
Comment (zrisher) [inline Data/CubeBlocks.sbc:30]: Please use 4 space indents in the XML files.

File: Data/CubeBlocks.sbc [hunk 4]
```diff
@@ -27,20 +27,15 @@
...
+				<Model BuildPercentUpperBound="0.33" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr1.mwm" />
+				<Model BuildPercentUpperBound="0.67" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr2.mwm" />
+				<Model BuildPercentUpperBound="1.00" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr3.mwm" />
+			</BuildProgressModels>
+			<MountPoints>
+				<MountPoint Side="Bottom" StartX="0.03" StartY="0.03" EndX="0.97" EndY="0.97" />
```
Comment (zrisher) [inline Data/CubeBlocks.sbc:36]: I know nothing about mountpoints, so hopefully you can enlighten me here. What's the reason for using a different Start and End attributes (`.03` and `.97` vs `.1` and `.9`) for the bottom mountpoint? And does anyone think removing the other mountpoints from the sides of the base is too restrictive?

File: Data/CubeBlocks.sbc [hunk 5]
```diff
@@ -695,4 +694,4 @@
     <!-- End Stations -->
 
   </CubeBlocks>
-</Definitions>
+</Definitions>
```
Comment (zrisher) [inline Data/CubeBlocks.sbc]: Please include a trailing newline.

**Comment by ghost** [comment]
All of that is done by Blender, I don't edit anything by hand.

**Comment by zrisher** [comment]
This has been merged and fixed up, but not released to public yet as I'm not sure if it will remove pre-existing Unlicensed classifiers because we're changing blocktype.

**Comment by Devlah** [comment]
From what I've been able to learn in my short time exploring the wide world of modding.  If the model of the classifier has been changed (as I recall it was, nice work  BTW) those mount points may have been adjusted in blender to more appropriately match the new model geometry.

**Comment by zrisher** [comment]
Yeah, if you click "files changed" on this commit you can see this significantly decreases the number of mount points for that block. If there's nothing else standing in the way of backwards compatibility, I may adjust them temporarily. But  the new mount point makes the most sense with which sides the model is flush with.

**Comment by ghost** [comment]
I think the next design will something that can be placed much easier but will still have a large area with the icon on it.

**Comment by zrisher** [comment]
@Souper07 Do you think the visual mountpoints on your next design will be the same as the original beacon? If so, I'll definitely go ahead and revert to the original mountpoints now.

**Comment by zrisher** [comment]
So I tested tonight building some blocks with the old mount points, then loading up the new version of GC and seeing if they broke off their grids. To my surprise, they stayed attached! Looks like once it's mounted the mount points don't play into whether it stays on the grid. So this is good to push.

I also noticed the bottom of the model is bright purple : ) This can attach to a block that doesn't fully cover it (try a triangle corner, i.e.). So we should probably fix that for future versions @Souper07

**Comment by ghost** [comment]
Ha. The bottom should just be whatever color the block is painted. I'll look into that tonight.

**Pull request merged**